packages/atlas/src/components/_comments/CommentInput/CommentInput.styles.ts

```diff
@@ -0,0 +1,118 @@
+import { keyframes } from '@emotion/react'
+import styled from '@emotion/styled'
+import TextareaAutosize from 'react-textarea-autosize'
+
+import { cVar, sizes } from '@/styles'
+
+import { CommentRow } from '../CommentRow'
+
+const reactionBarAnimation = keyframes`
+  from {
+    transform: translateX(-50%) scaleX(50%);
+  }
+  to {
+    transform: translateX(100%) scaleX(50%);
+  }
+`
+
+export const StyledTextArea = styled(TextareaAutosize)`
+  display: inline-block;
+  background-color: ${cVar('colorBackgroundMuted')};
+  width: 100%;
+  min-height: 40px;
+  height: auto;
+  border: none;
+  resize: none;
+  caret-color: ${cVar('colorBackgroundPrimary')};
+  color: ${cVar('colorTextStrong')};
+  font: ${cVar('typographyDesktopT200')};
+  letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
+  text-transform: ${cVar('typographyDesktopT200TextTransform')};
+
+  ::placeholder {
+    color: ${cVar('colorTextMuted')};
+    font: ${cVar('typographyDesktopT200')};
+    letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT200TextTransform')};
+    transition: all ${cVar('animationTransitionFast')};
+  }
+`
+export const Border = styled.div<{ 'data-show': boolean; 'data-processing': boolean }>`
+  overflow-x: hidden;
+  width: 100%;
+  height: 1px;
+  background-color: ${cVar('colorCoreNeutral800Lighten')};
+  transition: ${cVar('animationTransitionFast')};
+  transition-delay: cVar('animationTimingFast');
+
+  &[data-show='true'] {
+    height: 2px;
+    background-color: ${cVar('colorBackgroundPrimary')} !important;
```

Comment:
I would avoid using `!important` as much as I can. I think we could avoid this by passing the prop to StyledCommentRow?